src/dom.ts

```diff
@@ -95,9 +95,6 @@ export class FrameExecutionContext extends js.ExecutionContext {
   }
 
   createDebugScript(options: { record?: boolean, console?: boolean }): Promise<js.JSHandle<DebugScript> | undefined> {
-    if (!helper.isDebugMode())
```

Comment:
This is just on now? Does the recorder need it for something?

Reply:
There is another check higher up the call chain